internal/pkg/agent/cmd/svc_update.go

```diff
@@ -0,0 +1,12 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Should this file be moved to `internal/pkg/agent/application/upgrade/service_update.go` so it lives along side `internal/pkg/agent/application/upgrade/service.go`?

Reply:
Yes I think it should.